src/stores/notificationStore.ts

```diff
@@ -0,0 +1,34 @@
+import {observable} from 'mobx';
+import timeouts from '../constants/timeouts';
+import {BaseStore, RootStore} from './index';
+
+class NotificationStore extends BaseStore {
+  @observable showNotification: boolean = false;
```

Comment:
my main comment is here

probably it's a good idea to have a list of notifications in store. The intention of store is kinda client-side database

now i see that NotificationStore acts like a model, not store

i would suggest to introduce NotificationModel and have a `notifications: NotificationModel` list in store